### DISCUSSION TITLE
Can someone confirm if I have a "OneDrive SharePoint Shared Library"?

### DISCUSSION THREAD

**Discussion by steveklett** [abraunegg/onedrive#3010] [Q&A] [answered]
Hello All,

I'm new to Linux and not responsible for the O365 administration for our little business so I have minimal understanding of OneDrive and SharePoint. I -think- what I want to Sync is a "OneDrive SharePoint Shared Library" but it doesn't exactly match the screen shot from https://github.com/abraunegg/onedrive/blob/master/docs/sharepoint-libraries.md

![image](https://github.com/user-attachments/assets/14f068c8-ad46-4042-a28f-0fb998cf5718)

Here's a screen shot of what I'm wanting to Sync. I'd appreciate it if anyone can confirm if this is indeed a "OneDrive SharePoint Shared Library"
![image](https://github.com/user-attachments/assets/c3e61ce2-6121-47f7-9256-9378f5a5fc79)

-Steve

**Comment by abraunegg** [comment, marked as answer]
@steveklett 
Yes they appear to be SharePoint Libraries
